brickBreaker.js

```diff
@@ -0,0 +1,166 @@
+class BrickBreaker {
+  constructor() {
+    this.canvas = document.getElementById("canvas");
+    this.ctx = this.canvas.getContext("2d");
+    this.score = 0;
+    this.leftArrowPressed = false;
+    this.rightArrowPressed = false;
+    this.initializeEntities();
+    this.initializeEventListeners();
+  }
+
+  initializeEntities() {
+    this.ball = new Ball(this.ctx, this.canvas.width / 2, 200, 10, "#FFF", 4);
+    this.paddle = new Paddle(this.ctx, 100, 10, 150, 525, 5, "grey");
+    this.bricks = [];
+
+    const colors = ["red", "blue", "green", "yellow", "purple"];
+
+    let y = 0;
+    for (let row = 0; row < 5; row++) {
```

Comment:
number of rows and columns of bricks should be a constant in the game, defined in brickbreaker constructor above